### ISSUE TITLE
[Enchantment] no option to suppress warnings in intellij

### ISSUE THREAD

**Issue by DetachHead** [dart-code-checker/dart-code-metrics#806]
* **Dart code metrics version:** 4.14.0
* **Dart sdk version:** 2.16.2

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```yaml
dart_code_metrics:
  rules:
    - avoid-dynamic
```

</details>

**What did you do? Please include the source code example causing the issue.**

```dart
void foo(final dynamic a) {}
```

**What did you expect to happen?**
option to ignore the error like with the built in analyzer rules
![image](https://user-images.githubusercontent.com/57028336/163777447-292e3c23-5f7d-4240-9a38-387609d3c143.png)

**What actually happened?**
![image](https://user-images.githubusercontent.com/57028336/163777774-2e1df62c-5505-44be-9b64-44e02cfcb035.png)

**Comment by DetachHead** [comment]
workaround:  you can manually add a `//ignore:` comment, just like with the built in analyzer:
```dart
// ignore: avoid-dynamic
void foo(final dynamic a) {}
```

**Comment by incendial** [comment]
Available in DCM for Teams version https://dcm.dev/for-teams
